packages/devtools/lib/src/inspector/inspector_tree_canvas.dart

```diff
@@ -172,6 +172,8 @@ class InspectorTreeNodeCanvasRender
   // TODO(jacobr): share this method with test code.
   @override
   Icon hitTest(Offset location) {
+    if (offset == null) return null;
```

Comment:
I don't have a preference either way, but it seems we have a mix of inlined returns and returns in brackets throughout the codebase. We should probably pick a style and stay consistent with it.

Reply:
Lets follow the dart style guide which supports this sort of style for returns. We should cleanup other cases to match this style where it makes sense.
In general, only trivial returns should be on their own line and typically they should only occur near the top of methods so the control flow isn't confusing.